hack/fix_catalog_deployment_swagger

```diff
@@ -56,8 +56,22 @@ def rename_deployment_models(swagger):
         replace_value(swagger, "$ref", oldRef, newRef)
 
 
+def remove_global_security(swagger):
```

Comment:
Can we add some detail about why are we adding this hack and when can this be removed? Is there any bug/issue that is opened for the API and all such information.

Reply:
Added a comment at https://github.com/vmware/vra-sdk-go/blob/29a153c07e336ee1dd41699a346c0308e73828ff/hack/fix_catalog_deployment_swagger#L64